internal/backend/remote-state/s3/backend.go

```diff
@@ -674,6 +717,27 @@ func configureAssumeRole(obj cty.Value) *awsbase.AssumeRole {
 	return &assumeRole
 }
 
+func configureAssumeRoleWithWebIdentity(obj cty.Value) *awsbase.AssumeRoleWithWebIdentity {
+	cfg := &awsbase.AssumeRoleWithWebIdentity{
+		RoleARN:              stringAttrDefaultEnvVar(obj, "role_arn", "AWS_ROLE_ARN"),
+		Policy:               stringAttr(obj, "policy"),
```

Comment:
I guess if the value is not provided here, it then defaults to an empty string. Is the empty string a valid input to this call?

Reply:
I believe it's fine. An empty string is the same value as the zero value for the `Policy` field.